drivers/include/lis2dh12.h

```diff
@@ -114,13 +115,109 @@ enum {
     LIS2DH12_OK    =  0,            /**< everything was fine */
     LIS2DH12_NOBUS = -1,            /**< bus interface error */
     LIS2DH12_NODEV = -2,            /**< unable to talk to device */
+    LIS2DH12_NOINT = -3,            /**< wrong interrupt line (has to be 1 or 2) */
+    LIS2DH12_NODATA= -4,            /**< no data available */
+};
+
+/**
+ * @brief   Interrupt config register values
+ */
+enum {
+    LIS2DH12_INT_CFG_XLIE = 0x01,   /**< enable X low evnt */
+    LIS2DH12_INT_CFG_XHIE = 0x02,   /**< enable X high event */
+    LIS2DH12_INT_CFG_YLIE = 0x04,   /**< enable Y low event */
+    LIS2DH12_INT_CFG_YHIE = 0x08,   /**< enable Y high event */
+    LIS2DH12_INT_CFG_ZLIE = 0x10,   /**< enable Z low event */
+    LIS2DH12_INT_CFG_ZHIE = 0x20,   /**< enable Z high event */
+    LIS2DH12_INT_CFG_6D   = 0x40,   /**< enable 6-direction detection */
+    LIS2DH12_INT_CFG_AOI  = 0x80,   /**< and/or combination interrupt events */
 };
 
+/**
+ * @brief   Interrupt type values
+ */
+enum {
+    /* for interrupt 1 (CTRL_REG3) */
+    LIS2DH12_INT_1_TYPE_OVERRUN = 0x02, /**< FIFO overrun interrupt on INT_1 */
```

Comment:
I suggest to use
```suggestion
    LIS2DH12_INT1_TYPE_OVERRUN = 0x02, /**< FIFO overrun interrupt on INT1 */
```
here and below. The datasheet labels the pins as `INT1` and pin `INT2`. Also all interrupt related registers are named `INT1` and `INT2` even though the interrupt types are rather called `I1` and `I2`.